lib/cirro_io/client/base.rb

```diff
@@ -28,6 +28,26 @@ def self.custom_connection
           conn.use JsonApiClient::Middleware::Status, {}
         end
       end
+
+      # MonkeyPatch: https://github.com/JsonApiClient/json_api_client/issues/390
```

Comment:
```suggestion
      # HACK: https://github.com/JsonApiClient/json_api_client/issues/390
```

Just for better visibility.